react/features/conference/components/web/Subject.js

```diff
@@ -73,8 +171,11 @@ class Subject extends Component<Props> {
  */
 function _mapStateToProps(state) {
     const participantCount = getParticipantCount(state);
+    const domainSpeaker = getDominantSpeaker(state);
 
     return {
+        id: domainSpeaker && domainSpeaker.id,
```

Comment:
also can be refact with optional chaning
```suggestion
        id: domainSpeaker?.id,
```

Reply:
I think it can be even
```suggestion
        id: domainSpeaker ? domainSpeaker.id : ''
```
according to the data type check https://github.com/aeternity/jitsi-meet/blob/d4a0fae1e5124f0e609c7ce499ba54fbf4c719a2/react/features/conference/components/web/Subject.js#L39-L42

Reply:
or even with [nullish coalescing](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) also according to the [line 42](https://github.com/aeternity/jitsi-meet/blob/d4a0fae1e5124f0e609c7ce499ba54fbf4c719a2/react/features/conference/components/web/Subject.js#L42_)
```suggestion
        id: domainSpeaker.id ?? '',
```